src/settings/urlSettings.ts

```diff
@@ -25,30 +25,28 @@ export const ACCOUNT_ROOT_URL = "/account";
 export const ADMIN_V1_ROOT_URL = "/a1";
 export const ADMIN_V3_ROOT_URL = "/admin";
 export const ENTER_ROOT_URL = "/enter";
-export const ENTRANCE_BASE_URL = "/e";
-export const LOGIN_BASE_URL = `/login`;
+const ENTRANCE_BASE_URL = "/e";
+const LOGIN_BASE_URL = `/login`;
```

Comment:
Unused outside this module. I want to tighten everything up so that we use URL constants in the URL utility methods and/or the routers. We shouldn't be doing URL string interpolation in components.

Reply:
> Unused outside this module. I want to tighten everything up so that we use URL constants in the URL utility methods and/or the routers.

This was initially my approach months ago until I was reminded it wasn't up to the common coding standard. Is it changed now?

Reply:
I can't say I'd seen this documented as a standard anywhere. Exporting stuff when it's not desirable to use outside seems like an anti-pattern to me